cmd/submit.go

```diff
@@ -130,7 +130,7 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 		return err
 	}
 
-	sx, err := workspace.NewSolutions(dirs)
+	sx, err := workspace.NewMetadataCollection(dirs)
```

Comment:
Lets call this one collection like you do in the tests.

Reply:
Oh, yes! I agree.